python/instrumentation/openinference-instrumentation-langchain/examples/gemini_tracing.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from langchain_google_genai import ChatGoogleGenerativeAI
+from phoenix.otel import register
+
+# Set Environment Variables
+os.environ["GEMINI_API_KEY"] = ""
+os.environ["PHOENIX_CLIENT_HEADERS"] = "api_key="
+os.environ["PHOENIX_COLLECTOR_ENDPOINT"] = "https://app.phoenix.arize.com"
```

Comment:
I would probably not hard code these. users of OI probably don't necessarily use cloud